proto/brpb.proto

```diff
@@ -15,6 +15,30 @@ option (rustproto.lite_runtime_all) = true;
 
 option java_package = "org.tikv.kvproto";
 
+enum PrepareSnapshotBackupRequestType {
+    UpdateLease = 0;
```

Comment:
what's this used for?

Reply:
Updating the TTL, I will add some comments then.